modules/openy_gc_auth/modules/openy_gc_auth_custom/config/install/openy_gc_auth.provider.custom.yml

```diff
@@ -1,2 +1,7 @@
-enable_recaptcha: 1
+enable_recaptcha: 0
```

Comment:
are we going to disable captcha for current sites via hood_update_N?

Reply:
In this case ReCaptcha disabled for builds, without sitekey it not works

Reply:
So users would need to enable it manually after installation, right?
I guess instead of changing defaults to 'no captcha' it's better to tune build system to disable captcha via config set . Defaults should be production ready, not build ready

Reply:
I agree, reverted this to the previous state. By default on builds we have dummy plugin, so I think no need to tune the build system now.